File: RadRichTextEditor/CustomMergeField/CustomMergeFieldCS/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CustomMergeField
{
    public class Customer
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }

        public bool HasOrders { get; set; }
        public List<Order> Orders { get; set; }
    }
}

File: RadRichTextEditor/CustomMergeField/CustomMergeFieldVB/Customer.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace CustomMergeField
	Public Class Customer
		Public Property FirstName() As String
		Public Property LastName() As String

		Public Property HasOrders() As Boolean
		Public Property Orders() As List(Of Order)
	End Class
End Namespace
